frontend/ui/widget/naturallightwidget.lua

```diff
@@ -0,0 +1,433 @@
+    self.green_offset = self:adaptableNumber(self.powerd.fl.green_offset, 1)
+    self.exponent = self:adaptableNumber(self.powerd.fl.exponent, 0.1)
+
+    local separator = HorizontalSpan:new{ width = Size.span.horizontal_default }
+    local vspan = VerticalSpan:new{ width = 2 * Size.span.vertical_large }
+    local vertical_group = VerticalGroup:new{ align = "center" }
+    local title_group = HorizontalGroup:new{ align = "center" }
+    local white_group = HorizontalGroup:new{ align = "center" }
+    local red_group = HorizontalGroup:new{ align = "center" }
+    local green_group = HorizontalGroup:new{ align = "center" }
+    local exponent_group = HorizontalGroup:new{ align = "center" }
+    local button_group = HorizontalGroup:new{ align = "center" }
+    local text_gain = TextBoxWidget:new{
+        text = _("Gain"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.textbox_width + 2 * self.button_width
+    }
+    local text_offset = TextBoxWidget:new{
+        text = _("Offset"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.textbox_width + self.button_width
+    }
+    local text_white = TextBoxWidget:new{
+        text = _("White"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.text_width
+    }
+    local text_red = TextBoxWidget:new{
+        text = _("Red"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.text_width
+    }
+    local text_green = TextBoxWidget:new{
+        text = _("Green"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.text_width
+    }
+    local text_exponent = TextBoxWidget:new{
+        text = _("Exponent"),
+        face = self.medium_font_face,
+        bold = true,
+        alignment = "left",
+        width = self.text_width
+    }
+    local button_defaults = Button:new{
+        text = "Restore Defaults",
+        margin = Size.margin.small,
+        radius = 0,
+        width = self.width * 0.35,
+        show_parent = self,
+        callback = function()
+            self:setAllValues({white_gain = 25,
+                               white_offset = -25,
+                               red_gain = 24,
+                               red_offset = 0,
+                               green_gain = 24,
+                               green_offset = -65,
+                               exponent = 0.25})
+        end,
+    }
+    local button_cancel = Button:new{
+        text = "Cancel",
+        margin = Size.margin.small,
+        radius = 0,
+        width = self.width * 0.2,
+        show_parent = self,
+        callback = function()
+            self:setAllValues(self.old_values)
+            self:onClose()
+        end,
+    }
+    local button_ok = Button:new{
+        text = "OK",
```

Comment:
Is there anything better we can say here? E.g., `Save` or `Save settings`. See the UX guidelines here: http://koreader.rocks/doc/modules/ui.widget.confirmbox.html (not a requirement but nice to have)